indexer-cli: docker image is broken

Hi, I am trying to run indexer-cli from the official image but receiving an error when trying to launch cli:

```
# docker run -it --rm --entrypoint='' ghcr.io/graphprotocol/indexer-cli:latest bash
graph@b13dd0730917:~$ graph
/opt/indexer/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: Cannot find module 'p-map'
Require stack:
- /opt/indexer/packages/indexer-common/dist/indexer-management/allocations.js
- /opt/indexer/packages/indexer-common/dist/indexer-management/index.js
- /opt/indexer/packages/indexer-common/dist/index.js
- /opt/indexer/packages/indexer-cli/dist/actions.js
- /opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js
- /opt/indexer/node_modules/gluegun/build/loaders/module-loader.js
- /opt/indexer/node_modules/gluegun/build/loaders/command-loader.js
- /opt/indexer/node_modules/gluegun/build/runtime/runtime.js
- /opt/indexer/node_modules/gluegun/build/domain/builder.js
- /opt/indexer/node_modules/gluegun/build/index.js
- /opt/indexer/packages/indexer-cli/dist/cli.js
- /opt/indexer/packages/indexer-cli/bin/graph-indexer
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/indexer/packages/indexer-common/dist/indexer-management/allocations.js:19:33)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/indexer/packages/indexer-common/dist/indexer-management/allocations.js',
    '/opt/indexer/packages/indexer-common/dist/indexer-management/index.js',
    '/opt/indexer/packages/indexer-common/dist/index.js',
    '/opt/indexer/packages/indexer-cli/dist/actions.js',
    '/opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/module-loader.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/command-loader.js',
    '/opt/indexer/node_modules/gluegun/build/runtime/runtime.js',
    '/opt/indexer/node_modules/gluegun/build/domain/builder.js',
    '/opt/indexer/node_modules/gluegun/build/index.js',
    '/opt/indexer/packages/indexer-cli/dist/cli.js',
    '/opt/indexer/packages/indexer-cli/bin/graph-indexer'
  ]
}
```

I've tried the following images:
```
ghcr.io/graphprotocol/indexer-cli:v0.19.3
ghcr.io/graphprotocol/indexer-cli:v0.20.0
ghcr.io/graphprotocol/indexer-cli:v0.20.1
ghcr.io/graphprotocol/indexer-cli:v0.20.2
ghcr.io/graphprotocol/indexer-cli:v0.20.3
ghcr.io/graphprotocol/indexer-cli:v0.20.4
```

v0.18.6 works as expected:
```
# docker run -it --rm --entrypoint='' ghcr.io/graphprotocol/indexer-cli:v0.18.6 bash
graph@74b4dba5e929:~$ graph

  Welcome to graph-indexer CLI version 0.18.5!
  Type graph-indexer --help to view common commands.
```

--- a/packages/indexer-common/package.json
+++ b/packages/indexer-common/package.json
@@ -39,6 +39,7 @@
     "morgan": "1.10.0",
     "ngeohash": "0.6.3",
     "p-filter": "2.1.0",
+    "p-map": "4.0.0",
     "p-reduce": "2.1.0",
     "p-retry": "4.6.1",
     "p-timeout": "4.1.0",

